### PR TITLE
[PMK- 7524][PMK-7514] feat: add release note new templates

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue: 

[PMK- 7524][PMK-7514] 

### Changes

- Add release note new templates: Azion Starter Kit + HTMX

### Additional links

n/a.

[PMK-7514]: https://aziontech.atlassian.net/browse/PMK-7514?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx [hunk 1]
```diff
@@ -7,6 +7,23 @@ permalink: /documentacao/produtos/release-notes/
...
 
+## 12 de abril, 2024
+
+### Marketplace
+
+#### Azion Templates
+
+<Badge variant="accent">Preview</Badge>
+
+Dois novos tenmplates estão agora disponíveis:
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx:18]:
```suggestion
Dois novos templates estão agora disponíveis:
```

Comment (GabrielAzion) [inline src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx:18]: "tenmplates"

File: src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx [hunk 2]
```diff
@@ -7,6 +7,23 @@ permalink: /documentacao/produtos/release-notes/
...
+
+- [Azion Starter Kit](/pt-br/documentacao/produtos/guias/azion-starter-kit/): esse template é ideal para dar os primeiros passos na Plataforma Edge da Azion. Com este template, você pode acelerar a criação de um stack de edge básico para explorar a plataforma e seus recursos, incluindo configurações de desempenho e segurança para aprimorar e proteger sua aplicação.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx:20]:
```suggestion
- [Azion Starter Kit](/pt-br/documentacao/produtos/guias/azion-starter-kit/): esse template é ideal para dar os primeiros passos na Plataforma de Edge da Azion. Com este template, você pode acelerar a criação de um stack de edge básico para explorar a plataforma e seus recursos, incluindo configurações de desempenho e segurança para aprimorar e proteger sua aplicação.
```

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx [hunk 3]
```diff
@@ -7,6 +7,23 @@ permalink: /documentacao/produtos/release-notes/
...
+- [HTMX Boilerplate](/pt-br/documentacao/produtos/guias/htmx-boilerplate/): o template facilita a implantação e teste de um exemplo mínimo da biblioteca HTMX na Plataforma de Edge da Azion
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx:21]:
```suggestion
- [HTMX Boilerplate](/pt-br/documentacao/produtos/guias/htmx-boilerplate/): o template facilita a implantação e teste de um exemplo básico da biblioteca HTMX na Plataforma de Edge da Azion
```

File: src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx [hunk 4]
```diff
@@ -7,6 +7,23 @@ permalink: /documentacao/produtos/release-notes/
...
+### Marketplace
+
+#### Azion Templates
+
+<Badge variant="accent">Preview</Badge>
+
+Dois novos templates estão agora disponíveis:
+
+- [Azion Starter Kit](/pt-br/documentacao/produtos/guias/azion-starter-kit/): esse template é ideal para dar os primeiros passos na Plataforma de Edge da Azion. Com este template, você pode acelerar a criação de um stack de edge básico para explorar a plataforma e seus recursos, incluindo configurações de desempenho e segurança para aprimorar e proteger sua aplicação.
+- [HTMX Boilerplate](/pt-br/documentacao/produtos/guias/htmx-boilerplate/): o template facilita a implantação e teste de um exemplo básico da biblioteca HTMX na Plataforma de Edge da Azion
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx:21]:
```suggestion
- [HTMX Boilerplate](/pt-br/documentacao/produtos/guias/htmx-boilerplate/): o template facilita a implantação e teste de um exemplo básico da biblioteca HTMX na Plataforma de Edge da Azion.
```

**Pull request merged**